tests/unittests/Foundation/NSMapTableTests.mm

```diff
@@ -80,12 +82,19 @@ void assertMapTablesEqual(NSMapTable* table1, NSMapTable* table2) {
 
     // Verify that the stored value is autonilled on last release
     [objValue release];
+    // Clear the autorelease pool
+    [pool release];
     ASSERT_EQ(nil, [mapTable objectForKey:objKey]);
 
     [objKey release];
 }
 
-TEST(NSMapTable, WeakKey) {
+// Note: Test disabled on OSX
+// https://developer.apple.com/library/mac/releasenotes/Foundation/RN-FoundationOlderNotes/
+// NSMapTable zeroing weak changes - "weak-to-strong NSMapTables are not currently recommended, as the strong values for weak keys
+// which get zero'd out do not get cleared away (and released) until/unless the map table resizes itself"
```

Comment:
Not a problem

---
In reply to: [73383769](https://github.com/Microsoft/WinObjC/pull/797#discussion_r73383769) [](ancestors = 73383769)